File: film-information-service-interface/src/main/java/com/michael/service/UserDescService.java
package com.michael.service;

import com.michael.pojo.User;
import com.michael.pojo.UserDesc;
import entity.PageResult;

import java.util.List;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface UserDescService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<UserDesc> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum, int pageSize);


	/**
	 * 修改
	 */
	public void update(UserDesc user);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public UserDesc findOne(Long id);
	


	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(UserDesc user, int pageNum, int pageSize);
	
}

File: film-information-backstage-web/src/main/webapp/js/service/userDescService.js
//服务层
app.service('userDescService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../userDesc/findAll.do');
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../userDesc/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../userDesc/findOne.do?id='+id);
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../userDesc/update.do',entity );
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../userDesc/search.do?page='+page+"&rows="+rows, searchEntity);
	}    	
});
